Patterns/Dutch/Dutch-Numbers.yaml

```diff
@@ -80,15 +80,15 @@ FractionNotationWithSpacesRegex: !simpleRegex
 FractionNotationRegex: !simpleRegex         
   def: (((?<=\W|^)-\s*)|(?<![/-])(?<=\b))\d+[/]\d+(?=(\b[^/]|$))
 FractionUnitsRegex: !simpleRegex
-  def: ((?<onehalf>anderhalve|anderhalf)|(?<quarter>driekwart)|half|halve|helft|kwart)
+  def: ((?<onehalf>anderhalve|anderhalf)|(?<quarter>driekwart)|half|halve|helft|kwart|kwartier)
```

Comment:
Will this not cause issues with the NumberParser where it will parse kwartier as 0,25? Kwartier should only be used in a DateTime context. Kwart is 0,25 in a number context.

Reply:
removed 'kwartier' from Numbers.
This required a small change in BaseDurationParser in order to parse 'een kwartier' correctly as '0.25' instead of '1.25'.